internal/controller/nats/provisioner.go

```diff
@@ -19,6 +21,7 @@ func (r *Reconciler) handleNATSReconcile(ctx context.Context,
 
 	// set status to processing
 	nats.Status.Initialize()
+	events.Normal(r.recorder, nats, events.ReasonProcessing, "NATS resources are being initialized.")
```

Comment:
Maybe the message should be `Started processing NATS resource`? The reason is it an event and for a singular resource.

Reply:
Changed it to singular